components/clarinet-files/src/lib.rs

```diff
@@ -257,10 +241,63 @@ impl FileLocation {
         }
     }
 
-    pub fn get_project_manifest_location(&self) -> Result<FileLocation, String> {
-        let mut project_root_location = self.get_project_root_location()?;
-        project_root_location.append_path("Clarinet.toml")?;
-        Ok(project_root_location)
+    pub async fn get_project_manifest_location(
+        &self,
+        file_accessor: Option<&Box<dyn FileAccessor>>,
```

Comment:
@lgalabru @obycode 
Not sure if it's a good idea to pass `file_accessor` here. wdyt?

Reply:
Ok I thing I need to revisit some stuff here.

The thing is that `get_project_root_location` calls url_exists https://github.com/hirosystems/clarinet/blob/61b0fdc4cec84fbb19451c6a9ece4e49ba2082a1/components/clarinet-files/src/lib.rs#L212, whichhas not been implemented, and won't be since it would need to be `async`, but making `get_project_root_location` async would lead to tons of changes.

So I probably want to delete `FileLocation::url_exists` and rework `get_project_root_location` but I'm not sure how yet